pytradfri/device.py

```diff
@@ -136,48 +144,41 @@ class LightControl:
     def __init__(self, device):
         self._device = device
 
-    @property
-    def lights(self):
-        """Return light objects of the light control."""
-        return [Light(self._device, i) for i in range(len(self.raw))]
+        self.can_set_mireds = None
+        self.can_set_color = None
+        self._mireds_range = (None, None)
```

Comment:
I'm thinking it may be easier to just set the upper and lower bounds directly, rather than creating an intermediary tuple to store it?